src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -382,26 +385,80 @@ function WorkspaceMembersPage({policyMembers, personalDetails, route, policy, se
         </View>
     );
 
+    const handleBulkAction = (action: WorkspaceMemberBulkActionType) => {
```

Comment:
Not sure if we need this function. Can we just directly call each specific function, e.g. `askForConfirmationToRemove` when we define each option in the options array?

Reply:
yeah, that makes sense - I'll fix it